core/modules/wmgr/bin/qservWmgr.py

```diff
@@ -73,13 +72,17 @@ def main():
         app.config.from_pyfile(args.configFile)
     else:
         app.config.from_envvar('WMGRCONFIG')
-    Config.init(app)
+    config.Config.init(app)
 
     # add few blueprints
     app.register_blueprint(dbMgr.dbService, url_prefix='/dbs')
     app.register_blueprint(procMgr.procService, url_prefix='/services')
     app.register_blueprint(xrdMgr.xrdService, url_prefix='/xrootd')
 
+    # check authentication before every request
+    authCheck = auth.Auth(app.config)
+    app.before_request(authCheck.checkAuth)
```

Comment:
app.before_request is probably appropriate since I'm assuming the entire application should require authentication. If that's not the case, and if parts may be exposed without authentication, it may be worthwhile to use an [authentication decorator](http://flask.pocoo.org/docs/0.10/patterns/viewdecorators/).